packages/fxa-admin-panel/README.md

```diff
@@ -14,7 +14,7 @@ Outside of local development, this application is protected by SSO, a VPN connec
 
 This service will automatically spin up when `yarn start` is [ran from the root directory](https://github.com/mozilla/fxa#getting-started). A small Express server serves a React application and exposes the server config file for the client to consume via a meta tag.
 
-The React dev server runs at [localhost:8092](http://localhost:8092/) which can be useful when building components if you'd like an auto page refresh on file changes, however, the Express server that serves the React application and proxies its static resources runs at [localhost:8091](http://localhost:8091/). Develop on `:8091` if you need access to anything set in the server configuration file, including the URI for connecting to the `fxa-admin-server`.
+The React dev server runs at [localhost:8091](http://localhost:8092/) which can be useful when building components if you'd like an auto page refresh on file changes, however, the Express server that serves the React application and proxies its static resources runs at [localhost:8091](http://localhost:8091/). Develop on `:8091` if you need access to anything set in the server configuration file, including the URI for connecting to the `fxa-admin-server`.
```

Comment:
The port number in the link need to be updated to match?

Reply:
I was having some trouble with startup and csp blocking things on admin panel. This was just cruft from trying to get that working better. I'll revert this change. I might revisit this and file another small PR.